doc/user_guide/psyclone_command.rst

```diff
@@ -317,6 +325,15 @@ The default behaviour may be changed by adding the
 :ref:`configuration file <config-default-section>`. Note that any
 command-line setting always takes precedence.
 
+Overriding Fortran Intrinsics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+PSyclone attempts to canonicalise Fortran Intrinsics, which involves adding
```

Comment:
This needs a bit of re-wording now.

Reply:
Rewritten now - the final sentence is inverted to be correct w.r.t the new argument, and the early part is rewritten for what PSyclone now does.